internal/observability/tracing/conf.go

```diff
@@ -11,17 +11,25 @@ import (
 const (
 	confKey        = "tracing"
 	jaegerExporter = "jaeger"
+	otlpExporter   = "otlp"
 )
 
 var (
 	errJaegerConfigUndefined   = errors.New("jaeger configuration is empty")
 	errJaegerEndpointUndefined = errors.New("jaeger endpoint undefined")
+
+	errOTLPConfigUndefined   = errors.New("otlp configuration is empty")
+	errOTLPEndpointUndefined = errors.New("otlp endpoint undefined")
 )
 
 // Conf is optional configuration for tracing.
 type Conf struct {
+	// ServiceName is the name of the service reproted to the exporter.
+	ServiceName *string
```

Comment:
I think this should have the `yaml` and `conf` struct tags

Reply:
Also, there is a typo in the comment. `reproted ` should be `reported.